src/Blueprint.php

```diff
@@ -136,4 +138,68 @@ protected function shouldGenerate(array $types, array $only, array $skip): bool
 
         return true;
     }
+
+    private function transformPropertiesToArray(string $content, array $properties): string
+    {
+        preg_match_all('/controllers:(?:\n|.)*$/m', $content, $controllers);
```

Comment:
Simply find indented properties at the beginning of the line and add a numeric suffix.